oggopusEncoder.js

```diff
@@ -78,7 +78,9 @@ OggOpusEncoder.prototype.encode = function( buffers ) {
 };
 
 OggOpusEncoder.prototype.encodeFinalFrame = function() {
-  this.encode( new Float32Array( this.encoderBufferLength - this.encoderBufferIndex ) );
+  this.encode(new Array(this.numberOfChannels).fill().map(function() {
+    return Float32Array(this.encoderBufferLength - this.encoderBufferIndex);
```

Comment:
Yes we do, I can't believe how sloppy I've been on this PR.
It works without for now but will be completed broken in ES6.